src/components/Button/Button.scss

```diff
@@ -71,6 +71,38 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
   }
 }
 
+.globalTheming.inSegmentedGroup,
```

Comment:
Once we remove the .globalTheming these can be safely removed as well. Specificity issue.